python/lsst/ts/m2gui/actuator_force.py

```diff
@@ -0,0 +1,68 @@
+# This file is part of ts_m2gui.
+#
+# Developed for the LSST Data Management System.
+# This product includes software developed by the LSST Project
+# (https://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+__all__ = ["ActuatorForce"]
+
+from dataclasses import dataclass, field
+
+from . import NUM_ACTUATOR
+
+
+@dataclass
+class ActuatorForce:
```

Comment:
I thought this as well in the initial beginning of this PR. Then, I realized this might make the code complex because the received packet from M2 cell is a JSON packet. Each field will come as a `list`. If I use the `ActuatorForce`, I will need to use one or more `for` loops to put all data to all `ActuatorForce` instances. Therefore, I ended to use the `list` directly for each field to make it simple. But I will keep your suggestion in mind if it ends that this is needed. Thanks!